### PR TITLE
Ptb dataset jeremyasapp

### PR DESCRIPTION
**Pull request by jeremyasapp**

### CODE REVIEW

File: flambe/tokenizer/word.py [hunk 1]
```diff
@@ -10,6 +10,27 @@
...
+        Returns
+        -------
+        List[str]
+            The output word tokens, as a list of strings
+
+        """
+        return example.split()
+
+
+class NLTKWordTokenizer(Tokenizer):
```
Comment (iitzco-asapp) [inline flambe/tokenizer/word.py]: 👍

File: tests/unit/nlp/language_modeling/test_lm_datasets.py [hunk 2]
```diff
@@ -1,13 +1,23 @@
-from flambe.nlp.language_modeling import PTBDataset
+# TODO: uncomment after fixing circleCI
```
Comment (iitzco-asapp) [inline tests/unit/nlp/language_modeling/test_lm_datasets.py]: Lets fix this before merging imo. I can help investigate.

File: flambe/metric/dev/bpc.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import torch
+
+from flambe.metric import Metric
+
+
+class BPC(Metric):
```
Comment (iitzco-asapp) [inline flambe/metric/dev/bpc.py]: Lets add also tests for this metric!

File: tests/unit/nlp/language_modeling/test_lm_datasets.py (hunk 2 above)
Comment (jeremyasapp) [reply tests/unit/nlp/language_modeling/test_lm_datasets.py]: I don't know how to. You can try

File: flambe/metric/dev/bpc.py (hunk 3 above)
Comment (jeremyasapp) [reply flambe/metric/dev/bpc.py]: I guess i can add one or two
